plugin/pulp/plugin/models/importer.py

```diff
@@ -19,18 +18,17 @@ class Importer(PlatformImporter):
     additional persistent importer data by subclassing this object and adding Django fields. We
     defer to the Django docs on extending this model definition with additional fields.
 
-    Validation is done the Django way, so custom validation can also be added the same as any
-    Django model. We defer to the Django docs on adding custom validation to the subclassed
-    importer. If any of the model validation methods are overridden, be sure to call super() so the
-    platform can still perform its validation.
+    Validation at the API level is handled by plugin specific subclass of
```

Comment:
s/Validation at the API level is handled by/Validation of the Importer is done at the API by the/